### ISSUE TITLE
James' graphs stuck at 72dpi

### ISSUE THREAD

**Issue by triangle-man** [decc/calculator-book#1]
TL;DR: short-term fix: could you rm those pngs that come from .dot files in the /figures directory on the server? (Watch out -- some pngs didn't come from .dots).

Long description: Figures for Appendix 10 are produced as png from the .dot files in figures/ by way of a Makefile. I've updated the Makefile to create figures at 300dpi rather than the default 72dpi, however the old versions still exist on the server and therefore `make` doesn't make the new ones.

Ideally one would do `rm figures/*.png` in a script -- but there are other pngs in that directory which don't come from .dots. Best long-term solution is presumably to move all the .dots somewhere else.
